### PR TITLE
Validate datamanager package

### PR DESCRIPTION
**Pull request by rsekijima**
[AUTOMLALS-183](https://gohypergiant.atlassian.net/browse/AUTOMLALS-183)
Minimal validation of the datamanager package
Run all methods of DataManager class using local and aws providers

### CODE REVIEW

File: datamanager/__init__.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class DataManager(
 ):
     storage_provider: str = "local"
     volume_name: str = "/tmp"
+    volume_account: str = None
```
Comment (joshuacook) [inline datamanager/__init__.py]: Let's remove this field. We don't need it anymore.

File: datamanager/controllers/dataset.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 from pandas import read_excel as pd_read_excel
 from pandas import ExcelFile
 from pyarrow.lib import ArrowInvalid
-from dataclient.exceptions import DatasetError, DataRepoError, DataClientError
+from ..exceptions import DatasetError, DataRepoError, DataManagerError
```
Comment (joshuacook) [inline datamanager/controllers/dataset.py]: remove `DataRepoError`

File: datamanager/exceptions.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class DatasetError(Exception):
+    pass
+
+
+class DataRepoError(Exception):
```
Comment (joshuacook) [inline datamanager/exceptions.py]: remove this

**Review by joshuacook — APPROVED** [review]
Just a few changes then merge

File: datamanager/__init__.py (hunk 1 above)
Comment (rsekijima) [reply datamanager/__init__.py]: Removed and AWS is working fine without it

File: datamanager/controllers/dataset.py (hunk 2 above)
Comment (rsekijima) [reply datamanager/controllers/dataset.py]: removed

File: datamanager/exceptions.py (hunk 3 above)
Comment (rsekijima) [reply datamanager/exceptions.py]: removed

**Pull request merged**